data/Rishabh-Singh-Codes.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "Rishabh Singh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Non-Tech -> **Full Stack Developer** | Learner | Explorer ",
+   "tags": ["Open Source", "Javascript", "MERN", "Full Stack Developer"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Rishabh8Singh" },
+    { "icon": "FaGithub", "url": "https://github.com/Rishabh-Singh-Codes" },
```

Please remove the `,`
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/Rishabh-Singh-Codes" }
```